app/controllers/users_controller.rb

```diff
@@ -0,0 +1,49 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :update, :destroy]
+
+  def index
+    users = authorize User.where(admin: false)
+    render json: users, status: :ok
+  end
+
+  def show
+    render json: @user, status: :ok
+  end
+
+  def create
+    @user = authorize User.create(user_params)
+
+    if @user.errors.present?
```

Comment:
Quando for fazer isso aqui, tenta sempre usar o cenário de sucesso antes do cenário de erro...

```ruby
if @user.persisted?
  render json: @user, status: :created
else
  validation_error(@user)
end
```

Apesar que nesse caso é melhor vc fazer com rescue mesmo.